docker-compose.yml

```diff
@@ -118,14 +118,13 @@ services:
       - REACT_APP_APP_PORT=3006
       - REACT_APP_MENU_BASE_URL=http://localhost:8086
       - REACT_APP_MENU_URL=https://admin.dev.buerokratt.ee
-      - REACT_APP_RUUTER_V1_PRIVATE_API_URL=http://localhost:8086
-      - REACT_APP_RUUTER_V2_PRIVATE_API_URL=http://localhost:8086
       - REACT_APP_RUUTER_V2_SERVICE_API_URL=http://localhost:8086
       - REACT_APP_AUTH_BASE_URL=http://localhost:8086
       - REACT_APP_AUTH_PATH=/auth
       - REACT_APP_MENU_PATH=/chat/menu.json
       - REACT_APP_API_PATH=/
       - REACT_APP_LOCAL=true
+      - REACT_APP_RUUTER_PRIVATE_API_URL=http://localhost:8086
```

Comment:
Please modify it to be Without private or public

Reply:
Updated